assets/stylesheets/_vars.scss

```diff
@@ -15,7 +15,7 @@
 $guss-colours: $pasteup-palette;
 
 $c-brand: guss-colour(guardian-brand);
-$c-brand-mid: #005083;
+$c-brand-tint: #025b8c;
```

Comment:
Nope, frontend use a lightening function base on the brand colour. But I just hardcoded, might just try that.